examples/binaries/waiter/Cargo.toml

```diff
@@ -15,6 +15,9 @@ gstd.workspace = true
 gear-wasm-builder.workspace = true
 
 [dev-dependencies]
+gtest.workspace = true
+gear-core.workspace = true
```

Comment:
add issue to remove gear-core from here. its irrational to import/link/build crate here just for 1 wrapper across [u8; 32]

Reply:
Actually this is not about a wrapper, it is about the `MessageId` returned by the `RunResult::sent_message_id()` method. Should we instead add an issue that the method should return the `MessageId` type from the `gstd` crate instead?

Reply:
> Actually this is not about a wrapper, it is about the `MessageId` returned by the `RunResult::sent_message_id()` method. Should we instead add an issue that the method should return the `MessageId` type from the `gstd` crate instead?

maybe after introducing gstd-primitives crate